prow/images/test-runner/Dockerfile

```diff
@@ -34,9 +33,14 @@ RUN go get -u github.com/google/ko/cmd/ko
 RUN go get -u github.com/golang/dep/cmd/dep
 RUN go get -u github.com/google/licenseclassifier
 RUN go get -u github.com/jstemmer/go-junit-report
-RUN go get -u github.com/raviqqe/liche
+RUN GO111MODULE="on" go get -u github.com/raviqqe/liche
```

Comment:
Nice! I was trying to update the test-runner to go 1.13 and gave up when I ran into this `liche` install error